### PR TITLE
[ES] added content/es/shift-left.md

### PR DESCRIPTION
**Pull request by ramrodo**
### Describe your changes

Added `es/shift-left.md`.

This PR includes changes/work from this other stale PR: https://github.com/cncf/glossary/pull/1871

### Related issue number or link (ex: `resolves #issue-number`)

resolves https://github.com/cncf/glossary/issues/1773

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by electrocucaracha — APPROVED** [review]
/lgtm

File: content/es/shift-left.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Shift Left
+status: Completed
+category: Concepto
+tags: ["methodology", "", ""]
+---
+
+## ¿Qué es?
+
+Left en Shift Left se refiere a las etapas anteriores en un ciclo de vida de desarrollo de software,
```
Comment (krol3) [inline content/es/shift-left.md:10]:
```suggestion
El desplazamiento a la izquierda se refiere a las etapas anteriores en un ciclo de vida de desarrollo de software,
```

**Review by krol3 — COMMENTED** [review]
Traduzir el termino al espanol y despues de explicar lo que significa en ingles, podemos usarlo en el texto

File: content/es/shift-left.md (hunk 1 above)
Comment (ramrodo) [reply content/es/shift-left.md:10]:
```suggestion
El Desplazamiento a la Izquierda (Shift Left) se refiere a las etapas anteriores en un ciclo de vida de desarrollo de software,
```

**Review by krol3 — APPROVED** [review]
LGTM 🇲🇽 Rodolfo

**Pull request merged**